NLog.Web.AspNetCore/LayoutRenderers/AspNetRequestFormLayoutRenderer.cs

```diff
@@ -0,0 +1,88 @@
+﻿#if ASP_NET_CORE
+using Microsoft.Extensions.Primitives;
+#endif
+using NLog.LayoutRenderers;
+using NLog.Web.Internal;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+
+namespace NLog.Web.LayoutRenderers
+{
+    /// <summary>
+    /// ASP.NET Request Form Data
+    /// </summary>
+    /// <example>
+    /// <para>Example usage of ${aspnet-request-form}:</para>
+    /// <code lang="NLog Layout Renderer">
+    /// ${aspnet-request-form} - Produces - All Form Data from the Request with each key/value pair separated by a new line.
+    /// ${aspnet-request-form:Include=id,name} - Produces - Only Form Data from the Request with keys "id" and "name".
+    /// ${aspnet-request-form:Exclude=id,name} - Produces - All Form Data from the Request except the keys "id" and "name".
+    /// ${aspnet-request-form:Separator= / } - Produces - All Form Data from the Request with each key/value pair separated by " / ".
+    /// </code>
+    /// </example>
+    [LayoutRenderer("aspnet-request-form")]
+    public class AspNetRequestFormLayoutRenderer : AspNetLayoutRendererBase
+    {
+        /// <summary>
+        /// Gets or sets the form keys to include in the output.  If omitted, all are included.
+        /// </summary>
+        /// <docgen category='Rendering Options' order='10' />
+        public string Include { get; set; }
+
+        /// <summary>
+        /// Gets or sets the form keys to exclude in the output.  If omitted, none are excluded.
+        /// </summary>
+        /// <docgen category='Rendering Options' order='10' />
+        public string Exclude { get; set; }
```

Comment:
Idem, Set please :)

Reply:
Sorry, I'm not sure what you are asking for on this one.

Reply:
the ordering of the comments are changed. I meant: https://github.com/NLog/NLog.Web/pull/313#discussion_r216109918